decks/service-worker.mdx

```diff
@@ -0,0 +1,147 @@
+import { Head } from 'mdx-deck'
+
+export { default as theme } from '../theme'
+
+<Head>
+  <title>Service Worker</title>
+  <meta name="author" content="Tobias Linkohr" />
+</Head>
+
+# Service Worker
+
+by Tobias Linkohr  
+([github](https://github.com/Tobi-mmt))
+
+
+---
+
+# What is it?
+programmable network proxy
+
+---
+
+![](./images/service-worker/sw.jpeg)
+
+---
+
+# what can we do ?
+
+- receive push notifications
+- use app offline
+- cache management
+
+---
+
+# Good to know
+
+- The Service Worker runs in its __own global script context__
+- It isn’t tied to a specific web page
+- It __can't__ access the DOM
+
+---
+
+# Lifecycle
+
+![](./images/service-worker/lifecycle.png)
```

Comment:
Could you change this to <img src="https://.." /> please. This is needed for the build.